src/hearings/components/hearing-actual-summary/hearing-actual-summary.component.ts

```diff
@@ -13,23 +14,36 @@ import * as fromHearingStore from '../../store';
 export class HearingActualSummaryComponent implements OnInit {
   @Input() public hearingState$: Observable<fromHearingStore.State>;
   @Input() public hearingActualsMainModel: HearingActualsMainModel;
+  @Input() public adjournReasons: LovRefDataModel[];
 
   public isCompleted: boolean;
   public isAdjourned: boolean;
+  public adjournReasonTypeValue: string;
   public isPaperHearing$: Observable<boolean>;
   public hearingDays: { actualHearingDay: ActualHearingDayModel; plannedHearingDay: PlannedHearingDayModel }[] = [];
   public dateFormat = HearingDateEnum;
   public answerSource = AnswerSource;
 
   public ngOnInit(): void {
-    this.isCompleted = this.hearingActualsMainModel.hearingActuals.hearingOutcome.hearingResult === HearingResult.COMPLETED;
-    this.isAdjourned = this.hearingActualsMainModel.hearingActuals.hearingOutcome.hearingResult === HearingResult.ADJOURNED;
+    const hearingOutcome = this.hearingActualsMainModel &&
+      this.hearingActualsMainModel.hearingActuals &&
+      this.hearingActualsMainModel.hearingActuals.hearingOutcome;
+    this.isCompleted = hearingOutcome && hearingOutcome.hearingResult === HearingResult.COMPLETED;
+    this.isAdjourned = hearingOutcome && hearingOutcome.hearingResult === HearingResult.ADJOURNED;
+    console.log('IS ADJOURNED', this.isAdjourned);
```

Comment:
@johnbenjamin-hmcts - please remove this `console.log` statement if no longer required, and the one below

Reply:
Thank you @Daniel-Lam I have removed the `console.log` statements.